### PR TITLE
Add iota-prom-exporter role

### PR DESCRIPTION
**Pull request by ender503**
The iota-prom-exporter is used to expose the IRI information metric for
Prometheus.

To install the iota-prom-exporter on the host which has IRI service,
add iota-prom-exporter role and use it in the monitoring.yml with
ansible-Playbook.yml.

### CODE REVIEW

File: roles/iota-prom-exporter/tasks/iota-prom-exporter.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+- name: set variables centos/redhat
```
Comment (wusyong) [inline roles/iota-prom-exporter/tasks/iota-prom-exporter.yml]: Do centos/redhat need to install dependency package? They are not using apt AFAIK.

File: roles/iota-prom-exporter/tasks/deps.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+- name: Install some packages
```
Comment (wusyong) [inline roles/iota-prom-exporter/tasks/deps.yml]: Rename to specific package type might be better
```suggestion
- name: Install dependency packages
```

Comment (ender503) [reply roles/iota-prom-exporter/tasks/deps.yml]: Fixed.

File: roles/iota-prom-exporter/tasks/iota-prom-exporter.yml (hunk 1 above)
Comment (ender503) [reply roles/iota-prom-exporter/tasks/iota-prom-exporter.yml]: No, they don't use APT, let me think how to revise this parts.

File: roles/iota-prom-exporter/tasks/deps_yum.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- name: Install dependency packages
+  yum:
```
Comment (ender503) [inline roles/iota-prom-exporter/tasks/deps_yum.yml]: Add the `deps_yum.yml` to install dependency packages on CentOS or Red Hat.

File: roles/iota-prom-exporter/tasks/deps_yum.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    name: "{{ packages }}"
+    update_cache: yes
+  vars:
+    packages:
+      - systemd-docker
```
Comment (ender503) [inline roles/iota-prom-exporter/tasks/deps_yum.yml]: But seems like the systemd-docker is not exist on the yum packages, let me think how to revise this part.

**Comment by ender503** [comment]
I

**Pull request closed without merging**